src/util-landlock.c

```diff
@@ -102,7 +103,13 @@ static inline struct landlock_ruleset* LandlockCreateRuleset()
         return NULL;
     }
     if (abi < 2) {
-        ruleset->attr.handled_access_fs &= ~LANDLOCK_ACCESS_FS_REFER;
+        if (FileStoreIsEnabled()) {
```

Comment:
you can use `RequiresFeature(FEATURE_OUTPUT_FILESTORE)` here instead

Reply:
Oh, nice!